syntax = "proto3";

import "google/api/annotations.proto";

import "common.proto";

message GetLinkCodeRecordRequest {
  string link_code = 1;
}

service Central {
  rpc GetLinkCodeRecord(GetLinkCodeRecordRequest) returns (LinkCodeRecord) {
	option (google.api.http) = { get: "/v1/linkcode/{link_code}" };
  }
}
